### ISSUE TITLE
bom to digikey script doesn't include all part numbers

### ISSUE THREAD

**Issue by nealjack** [lab11/eagle#10]
Only lists a single part in the part list, when many are identical:
![image](https://cloud.githubusercontent.com/assets/5566471/24060183/4691890a-0b0f-11e7-91fb-e6ca62bd394a.png)

**Comment by adkinsjd** [comment]
Yeah, I wouldn't use that unless you do eagle my way (which apparently is
no one). I'm probably going to remove it from the repo and assimilate to
everyone else
